rosdep/python.yaml

```
@@ -1385,6 +1385,21 @@ python-dt-apriltags-pip:
     pip: [dt-apriltags]
   ubuntu:
     pip: [dt-apriltags]
+python-duckdb-pip:
+  alpine:
+    pip: [duckdb]
+  arch:
+    pip: [duckdb]
+  debian:
+    pip: [duckdb]
+  fedora:
+    pip: [duckdb]
+  opensuse:
+    pip: [duckdb]
+  rhel:
+    pip: [duckdb]
+  ubuntu:
+    pip: [duckdb]
```

Nit: please use `'*'` to match all platforms:

```suggestion
  '*':
    pip: [duckdb]
```